Android/Nothing/app/quality.gradle

```diff
@@ -0,0 +1,15 @@
+detekt {
+    toolVersion = "1.14.1"
+    config = files("$rootProject.projectDir/config/detekt/detekt.yml")
+
+    reports {
+        xml {
+            enabled = true
+//            destination = file("$project.reporting.baseDir/detekt.xml")
+        }
+        html {
+            enabled = true
+//            destination = file("$project.reporting.baseDir/detekt.html")
```

Comment:
Same here

Reply:
This is currently used for local development. The report tells you what will break the build due to lack of quality